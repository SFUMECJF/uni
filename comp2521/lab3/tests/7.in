-15
-13
-14
-11
-9
-10
-12
-7
-5
-6
-3
-1
-2
-4
-8
-16
-17
-18
-22
-3341
