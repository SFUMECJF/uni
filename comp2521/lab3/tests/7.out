Tree:
            -15
            / \
           /   \
         -16   -13
         /     / \
       -17    /   \
       /    -14   -11
     -18          / \
     /           /   \
   -22         -12   -9
   /                 / \
-3341               /   \
                  -10   -7
                        / \
                       /   \
                      -8   -5
                           / \
                          /   \
                         -6   -3
                              / \
                             /   \
                            -4   -1
                                 /
                                -2

#nodes = 20,  #leaves = 8
Infix  : -3341 -22 -18 -17 -16 -15 -14 -13 -12 -11 -10 -9 -8 -7 -6 -5 -4 -3 -2 -1 
Prefix : -15 -16 -17 -18 -22 -3341 -13 -14 -11 -12 -9 -10 -7 -8 -5 -6 -3 -4 -1 -2 
Postfix: -3341 -22 -18 -17 -16 -14 -12 -10 -8 -6 -4 -2 -1 -3 -5 -7 -9 -11 -13 -15 
ByLevel: -15 -16 -13 -17 -14 -11 -18 -12 -9 -22 -10 -7 -3341 -8 -5 -6 -3 -4 -1 -2 
